src/main/java/com/arcbees/website/client/application/support/ScrollToAnimation.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
@meriouma @christiangoudreau Shouldn't this thing be a plugin? I see the ScrollToAnimation as something that we could need in every projects, and I find its current location wrong ... Should be more like a widget at least, no?